web/sites/default/config/core.entity_form_display.media.map.default.yml

```diff
@@ -39,90 +35,15 @@ content:
     third_party_settings: {  }
   field_geo_file:
```

Comment:
Not related to this line of code, but I had a thought: Should we make Geo file required on Geo map now, so people don't start creating maps again with only a preview image?

![image](https://user-images.githubusercontent.com/1520776/235009940-271d36f9-b8e7-4115-b1ad-ece9322e78cf.png)

Reply:
Good idea! Made the update